PostFreeSurfer/PostFreeSurferPipeline.sh

```diff
@@ -90,6 +90,24 @@ InflateExtraScale=`opts_DefaultOpt $InflateExtraScale 1`
 
 PipelineScripts=${HCPPIPEDIR_PostFS}
 
+verbose_red_echo "---> Starting Post FreeSurfer Pipeline"
+verbose_echo " "
+verbose_echo " Using parameters ..."
+verbose_echo "              --path: ${StudyFolder}"
+verbose_echo "           --subject: ${Subject}"
+verbose_echo "      --surfatlasdir: ${SurfaceAtlasDIR}"
+verbose_echo "  --grayordinatesdir: ${GrayordinatesSpaceDIR}"
+verbose_echo "  --grayordinatesres: ${GrayordinatesResolutions}"
+verbose_echo "         --hiresmesh: ${HighResMesh}"
+verbose_echo "        --lowresmesh: ${LowResMeshes}"
+verbose_echo " --subcortgraylabels: ${SubcorticalGrayLabels}"
+verbose_echo "  --freesurferlabels: ${FreeSurferLabels}"
+verbose_echo "     --refmyelinmaps: ${ReferenceMyelinMaps}"
+verbose_echo "           --mcsigma: ${CorrectionSigma}"
+verbose_echo "           --regname: ${RegName}"
+verbose_echo "   --PipelineScripts: ${PipelineScripts}"
```

Comment:
Unless you added this, `--PipelineScripts` is not an input argument.  And if it _is_ something that was added, it should be all lower case in the argument name for consistency.

Reply:
FWIW, I'm not sure if the longer term plan would be to switch to using `verbose_echo` more, but the other scripts mostly use `log_Msg` for parameter logging.

Reply:
@mharms `PipelineScripts` is derived from environment variable `HCPPIPEDIR_PostFS`. I changed the echo to reflect that. 

I've used verbose_echo where there were no `log_Msg` statements before.

Reply:
I don't think we need scripts to log things that are derived in a simple fashion from environment variables, or the values of any of the `HCPPIPEDIR_*` variables either.  If you have a specific reason to log `HCPPIPEDIR`, that might be reasonable.

Reply:
To be clear, I don't think `HCPPIPEDIR_PostFS` should ever be anything other than `${HCPPIPEDIR}/PostFreeSurfer/scripts`, and thus `HCPPIPEDIR` is a more succinct display of the only part of it that can be changed by the user.

Reply:
Along those lines, it seems like what we need in the PostFS script is a reporting of the HCPPIPEDIR, the HCPPipelines version, and other tool (FSL, Workbench) versions.  @coalsont can you suggest a script to model that after?  (I believe that you created some functions to simplify the process...?)

Reply:
Tim Brown may have made such things, I don't think I did.  I see only:

https://github.com/Washington-University/HCPpipelines/blob/master/global/scripts/fsl_version.shlib

Reply:
Also, I just noticed that that function puts the result into a variable with 'eval', when the right way to design it is for the function to echo the version number.  Since it is a function, it also shouldn't do an exit (a return, at most), and shouldn't use any `log_*` function since those output to stdout.

So no, there isn't a ready-made good way to report all that.  Just change the verbose echo to report HCPPIPEDIR and call it good enough for now.

Reply:
@coalsont @sivcek My goal, once we get all these PRs merged into the master, is that we do some cross-script harmonization, make sure we can easily get usage statements, that all the scripts report the tool and pipeline version in a consistent fashion, etc.  If you see/think of something along those lines, please add it to here https://github.com/Washington-University/HCPpipelines/issues/146 so that we don't lose track of it.

Reply:
@mharms Agreed. We have some solutions in Qu|Nex code and I added (and then pulled out) a bit in mppchecking.shlib (now with another name).